flowsa/data_source_scripts/EPA_NEI.py

```diff
@@ -208,26 +208,22 @@ def remove_duplicate_NEI_flows(df):
     return df
 
 
-def drop_GHGs(df):
+def drop_GHGs(df, *_):
     """
     GHGs are included in some NEI datasets. If these data are not
     compiled together with GHGRP, need to remove them as they will be
     tracked from a different source
     :param df: df, FBA format
     :return: df
     """""
-    # Flow names reflect source data prior to FEDEFL mapping, using 'FlowName'
-    # instead of 'Flowable'
     flowlist = [
         'Carbon Dioxide',
         'Methane',
         'Nitrous Oxide',
         'Sulfur Hexafluoride',
     ]
-
-    df = df.loc[~df['FlowName'].isin(flowlist)]
-
-    return df
+    flow_var = 'Flowable' if 'Flowable' in df.columns else 'FlowName'
```

Comment:
I'll need to confirm this is fine - we use Flowable after mapping, and FlowName before IIRC. There are syntax and capitalization issues that make this distinction important.

Reply:
Yes, that's correct. The goal of line 225 was to make the function suitable for use either before or after mapping.

Reply:
Right - but the flow names in the list above are specific to the NEI pre-mapping, as the names update after mapping.

Reply:
Ah, makes sense.

Reply:
Ah, makes sense.